src/Orchard.Web/Modules/Orchard.Widgets/Views/EditorTemplates/Parts.Widgets.WidgetPart.cshtml

```diff
@@ -24,6 +24,11 @@
 </fieldset>
 <fieldset>
     @Html.LabelFor(widget => widget.Name, T("Name"))
-    @Html.TextBoxFor(widget => widget.Name, new { @class = "text" })
+    @Html.TextBoxFor(widget => widget.Name, new {@class = "text"})
     <span class="hint">@T("The technical name of the Widget, used for css class and alternates.")</span>
 </fieldset>
+<fieldset>
+    @Html.LabelFor(widget => widget.CssClass, T("CSS Class"))
+    @Html.TextBoxFor(widget => widget.CssClass, new{@class = "text"})
+    <span class="hint">@T("The CSS classes of the Widget")</span>

```

Comment:
This is probably just me being a nitpick, but to be consistent, I think you should end hints/descriptions with periods.